pkg/controller/topology.go

```diff
@@ -65,6 +65,7 @@ func GenerateVolumeNodeAffinity(accessibleTopology []*csi.Topology) *v1.VolumeNo
 }
 
 func GenerateAccessibilityRequirements(
+	topologyKeys []string,
 	kubeClient kubernetes.Interface,
 	csiAPIClient csiclientset.Interface,
```

Comment:
`csiAPIClient` does not appear to be used any more in `GenerateAccessibilityRequirements`. Can it be removed?